src/Microsoft.AspNet.WebHooks.Custom.SqlStorage/WebHooks/SqlWebHookStore.cs

```diff
@@ -33,6 +37,24 @@ public SqlWebHookStore(SettingsDictionary settings, ILogger logger)
             CheckSqlStorageConnectionString(settings);
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="SqlWebHookStore"/> class with the given <paramref name="settings"/>
+        /// <paramref name="logger"/>, <paramref name="nameOrConnectionString"/>, <paramref name="schemaName"/> and <paramref name="tableName"/>. 
+        /// Using this constructor, the data will not be encrypted while persisted to the database.
+        /// </summary>
+        public SqlWebHookStore(SettingsDictionary settings, ILogger logger, string nameOrConnectionString, string schemaName, string tableName)
+            : base(logger)
+        {
+            if (settings == null)
+            {
+                throw new ArgumentNullException(nameof(settings));
+            }
+
+            _nameOrConnectionString = nameOrConnectionString;
```

Comment:
Sorry, just realized this and the other new constructor overload should call `CheckSqlStorageConnectionString(settings)` when `nameOrConnectionString==null`. Now using the longer overloads all the time in `HttpConfigurationExtensions`.